### PR TITLE
Stream homework

### PR DESCRIPTION
**Pull request by Evgenygroup**
#1

### CODE REVIEW

File: StreamAPI/BankAccount.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+
+public class BankAccount {
+    String IBANN;
```
Comment (sergeylukichev) [inline StreamAPI/BankAccount.java:3]: It is actually IBAN - International Bank Account Number.

File: StreamAPI/Main.java [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+            chars[i] = '*';
+        return new String(chars);
+    }
+
+
+    //6. method accepts a sentence and
+    // returns number of words starting with a certain letter
+    public static long numberOfWords(String input, String w) {
+        long count = Stream.of(input.split(" "))
+                .filter(s -> s.startsWith("a"))
```
Comment (sergeylukichev) [inline StreamAPI/Main.java:162]: I think it returns words, which start with 'a', not with `w`.

**Review by sergeylukichev — APPROVED** [review]

**Comment by sergeylukichev** [comment]
👍

File: StreamAPI/BankAccount.java (hunk 1 above)
Comment (Evgenygroup) [reply StreamAPI/BankAccount.java:3]: Yeah, of course IBAN .  Second N is extra)

File: StreamAPI/Main.java (hunk 2 above)
Comment (Evgenygroup) [reply StreamAPI/Main.java:162]: Exactly. Should be w there instead of  an "a".

**Pull request merged**